### ISSUE TITLE
Extracting OCRSearch

### ISSUE THREAD

**Issue by rahelarnold98** [vitrivr/cineast#235]
When extracting the feature [OCRSearch](https://github.com/vitrivr/cineast/blob/dev/cineast-core/src/main/java/org/vitrivr/cineast/core/features/OCRSearch.java), I always end up getting the attached error message. Is there a resource missing in [externalFiles.csv](https://github.com/vitrivr/cineast/blob/dev/externalFiles.csv) or is there an underlying bug?

```2021-11-24 14:24:25.983 [pool-4-thread-1] FATAL o.v.c.s.r.ExtractionTask - EXTRACTION ERROR in OCRSearch: for segmentID v_v_00001_1 
CvException [org.opencv.core.CvException: cv::Exception: OpenCV(4.5.1) /Users/runner/work/opencv/opencv/opencv-4.5.1/modules/dnn/src/caffe/caffe_io.cpp:1133: error: (-2:Unspecified error) FAILED: fs.is_open(). Can't open "resources/SceneTextExtractor/frozen_east_text_detection.pb" in function 'ReadProtoFromBinaryFile'
]
	at org.opencv.dnn.Dnn.readNetFromTensorflow_1(Native Method)
	at org.opencv.dnn.Dnn.readNetFromTensorflow(Dnn.java:183)
	at org.vitrivr.cineast.core.util.TextDetector_EAST.initialize(TextDetector_EAST.java:43)
	at org.vitrivr.cineast.core.util.TextDetector_EAST.initialize(TextDetector_EAST.java:53)
	at org.vitrivr.cineast.core.features.OCRSearch.processSegment(OCRSearch.java:265)
	at org.vitrivr.cineast.standalone.runtime.ExtractionTask.run(ExtractionTask.java:28)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)

**Comment by lucaro** [comment]
do you have `resources/SceneTextExtractor/frozen_east_text_detection.pb` locally?

**Comment by rahelarnold98** [comment]
No, I have only the files loaded by `./gradlew getExternalFiles` in my local resources directory.

**Comment by lucaro** [comment]
Hmm, that one is indeed missing from the external files. I will investigate where it ended up.

**Comment by lucaro** [comment]
The file should be now among the external ones, please try again.

**Comment by rahelarnold98** [comment]
The file is now downloaded into the resources directory, but unfortunately, into the wrong subfolder. The same is also true for `CRNN_VGG_BiLSTM_CTC.onnx`. Both of the files are stored within `resources/TextSpotter/` instead of in `resources/SceneTextExtractor/` as specified in `cineast-core/src/main/java/org/vitrivr/cineast/core/util/TextDetector_EAST.java`, respectively `cineast-core/src/main/java/org/vitrivr/cineast/core/util/TextRecognizer_CTC.java`.

**Comment by lucaro** [comment]
Thanks, I merged #236, so it should now be consistent.
